readme.md

```diff
@@ -26,7 +26,6 @@ When used with a kit, everything is available, according to that kit's configura
 - Font Awesome Pro
 - Font Awesome 6
 
```

Comment:
Looks like there are a couple of extra returns in here (not from you). Mind cleaning those up while you're in here?